scripts/performance_testing/README.md

```diff
@@ -0,0 +1,211 @@
+        Accept pipeline input?       false
+        Accept wildcard characters?  false
+
+    -ProcessResults [<SwitchParameter>]
+
+        Required?                    true
+        Position?                    named
+        Default value                False
+        Accept pipeline input?       false
+        Accept wildcard characters?  false
+
+    -ResultsFile <String>
+        Configures tool to process results.
+
+        Required?                    true
+        Position?                    named
+        Default value
+        Accept pipeline input?       false
+        Accept wildcard characters?  false
+
+    -ResultsDirectory <String>
+        Where results should be written to.
+
+        Required?                    false
+        Position?                    named
+        Default value                (Get-Location)
+        Accept pipeline input?       false
+        Accept wildcard characters?  false
+
+    -ReleaseTag <String>
+        The release tag to use for the test.
+
+        Required?                    false
+        Position?                    named
+        Default value                current
+        Accept pipeline input?       false
+        Accept wildcard characters?  false
+
+    -Suite <String>
+        Which suite to run.
+
+        Required?                    true
+        Position?                    named
+        Default value
+        Accept pipeline input?       false
+        Accept wildcard characters?  false
+
+    -Platform <String>
+        The platform to run on. This is just a descriptive string.
+
+        Required?                    false
+        Position?                    named
+        Default value                $PSVersionTable.Platform
+        Accept pipeline input?       false
+        Accept wildcard characters?  false
+
+    -Language <String>
+        The language to run on.
+
+        Required?                    true
+        Position?                    named
+        Default value
+        Accept pipeline input?       false
+        Accept wildcard characters?  false
+
+    <CommonParameters>
+        This cmdlet supports the common parameters: Verbose, Debug,
+        ErrorAction, ErrorVariable, WarningAction, WarningVariable,
+        OutBuffer, PipelineVariable, and OutVariable. For more information, see
+        about_CommonParameters (https://go.microsoft.com/fwlink/?LinkID=113216).
+
+INPUTS
+
+OUTPUTS
+
+
+RELATED LINKS
+
+```
+## Example Usage
+
+Run the `cert` suite for `c` from within the Coding Standards repository. 
+
+```
+.\scripts\performance_testing\Test-ReleasePerformance.ps1 -RunTests -DatabaseArchive ..\codeql-coding-standards-release-engineering\data\commaai-openpilot-72d1744d830bc249d8761a1d843a98fb0ced49fe-cpp.zip -Suite cert -Language c
+```
+
+Run the `cert` suite for `c` on an external release, specifying a `-ReleaseTag` as well. The `-ReleaseTag` parameter does not have to match the code you are testing, it is for organization purposes only. 
```

Comment:
I find this sentence a bit unclear:
The `-ReleaseTag` parameter does not have to match the code you are testing, it is for organization purposes only. 

code - meaning - src code being analyzed? or CodeQL query code version? maybe specify

Reply:
I added a clarification I hope it helps!

Reply:
ah ! yep, clearer now! :)